packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.ts

```diff
@@ -144,6 +144,19 @@ export default ({ api, coreSagas, networks }) => {
     )
   }
 
+  const defineCowboysCampaignGoal = function* (search) {
```

Comment:
Actually, I think we just keep this goal more generic and not cowboys related. We'll be able to use this to deep link to referrals for a lot of different use-cases.